server/routes/api.js

```diff
@@ -110,4 +109,46 @@ router.patch('/cards', (req, res) => {
     }
 })
 
+/*
+API call to get a single column from the neighborhood list using a constraint array.
+req: 1) the string name of the column to get data from.
+    2) a constraint_array of the same form as the api/getFilteredData endpoint.
+res: an array of numbers from the specified column with the specified constraints.
+*/
+router.get('/column', (req, res) => {
+    console.log("api/column call:")
+    console.log(req.body)
+
+    constraint_array = req.body.constraint_array;
+    column = req.body.column_name;
+
+    //determine the column number based on the column name
+    column_num = -1;
+    switch(column){
+		case "Median_House_Value": column_num = 1; break;
+		case "Median_Income": column_num = 2; break;
+		case "Median_Age": column_num = 3; break;
+		case "Tot_Rooms": column_num = 4; break;
+		case "Tot_Bedrooms": column_num = 5; break;
+		case "Population": column_num = 6; break;
+		case "Households": column_num = 7; break;
+		case "Latitude": column_num = 8; break;
+		case "Longitude": column_num = 9; break;
+		case "Distance_to_coast": column_num = 10; break;
+		case "Distance_to_LA": column_num = 11; break;
+		case "Distance_to_SanDiego": column_num = 12; break;
```

Comment:
"Distance_to_SD" for consistency, please also change the below to SJ and SF respectively.

Reply:
calling it "Distance_to_LA" and the rest as the full name is NOT a mistake. They are based on the name of columns from the csv file. the original CSV column has the naming scheme, and I want to keep it consistent with the CSV file.